readthedocs/oauth/models.py

```diff
@@ -206,3 +195,35 @@ def matches(self, user):
                 },
             ),
         } for project in projects]
+
+
+class RemoteRelation(models.Model):
+    remoterepository = models.ForeignKey(
+        RemoteRepository,
+        related_name='remote_relations',
+        on_delete=models.CASCADE
+    )
+    user = models.ForeignKey(
+        User,
+        related_name='remote_relations',
+        on_delete=models.CASCADE
+    )
+    account = models.ForeignKey(
+        SocialAccount,
+        verbose_name=_('Connected account'),
+        related_name='remote_relations',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+    )
+    active = models.BooleanField(_('Active'), default=False)
+    admin = models.BooleanField(_('Has admin privilege'), default=False)
+    json = JSONField(_('Serialized API response'))
+
+    pub_date = models.DateTimeField(_('Publication date'), auto_now_add=True)
+    modified_date = models.DateTimeField(_('Modified date'), auto_now=True)
```

Comment:
For new models, I prefer if we can use the `TimeStampedModel` from django extensions.

Reply:
Okay! I'll update the PR with ``TimeStampedModel``